tests/sentry/incidents/endpoints/test_organization_alert_rule_index.py

```diff
@@ -980,14 +980,14 @@ def test_status_and_date_triggered_sort_order(self):
         result = json.loads(response.content)
         assert len(result) == 7
         # Assert critical rule is first, warnings are next (sorted by triggered date), and issue rules are last.
-        [r["id"] for r in result] == [
-            str(alert_rule_critical.id),
-            str(another_alert_rule_warning.id),
-            str(alert_rule_warning.id),
-            str(self.yet_another_alert_rule.id),
-            str(self.other_alert_rule.id),
-            str(self.alert_rule.id),
-            str(self.issue_rule.id),
+        assert [r["id"] for r in result] == [
+            f"{alert_rule_critical.id}",
```

Comment:
This was cherrypicked from https://github.com/getsentry/sentry/pull/28014 so it'll have some fstring conversions, no big deal.